nodeprobe/nodeprobe.go

```diff
@@ -136,3 +136,27 @@ func (p *Prober) AddNode(name string, node Node) {
 
 	p.nodes[name] = node
 }
+
+func (p *Prober) CheckBeaconNodeHealth(ctx context.Context) error {
+	err := p.nodes["consensus client"].Healthy(ctx)
```

Comment:
it's not a part of your PR, but I think these string constants are error-prone, so I thought maybe it's worth to leave a TODO that we could e.g. add a method to clients that returns their name or solve this in another way